up-linux-streamer-plugin/Cargo.toml

```diff
@@ -44,14 +44,14 @@ serde = { version = "1.0.154" }
 serde_json = { version = "1.0.94" }
 tokio = { version = "1.35.1", default-features = false }
 up-rust = { workspace = true }
-up-transport-zenoh = { version = "0.1.1"  }
-up-transport-vsomeip = { git = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust.git", tag = "v0.1.0", default-features = false }
+up-transport-zenoh = { version = "0.3.0"  }
+up-transport-vsomeip = { git = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust.git", tag = "v0.3.0", default-features = false }
 up-streamer = { path = "../up-streamer" }
 usubscription-static-file = {path = "../utils/usubscription-static-file"}
-zenoh = { version = "1.0.0-alpha.6", features = ["unstable", "internal", "plugins"]}
-zenoh-core = { version = "1.0.0-alpha.6" }
-zenoh-plugin-trait = { version = "1.0.0-alpha.6" }
-zenoh-result = { version = "1.0.0-alpha.6" }
-zenoh-util = { version = "1.0.0-alpha.6" }
-zenoh_backend_traits = { version = "1.0.0-alpha.6" }
+zenoh = { version = "1.0.0", features = ["unstable", "internal", "plugins"] }
```

Comment:
see previous comment

Reply:
The plugins one I think we need.

But I'm thinking to generally only compile the reference implementations of the streamers conditionally going forward.

Will help with needing to compile vsomeip as well.

Reply:
Needed to keep the "plugins" feature, but again, was able to remove "internal" and "unstable".

Reply:
Updated to compile the reference implementations of the streamers in full conditionally based on a feature flag.